updating/updating-restricted-network-cluster/index.adoc

```diff
@@ -0,0 +1,23 @@
+:_content-type: ASSEMBLY
+[id="about-restricted-network-updates"]
+= About restricted network updates
+include::_attributes/common-attributes.adoc[]
+:context: about-restricted-network-updates
+
+toc::[]
+
+A restricted network environment is one in which your cluster nodes cannot access the internet. 
+For this reason, you must populate a registry with the installation images. 
+If your registry host cannot access both the internet and the cluster, you can mirror the images to a file system that is disconnected from that environment and then bring that host or removable media across that gap. 
+If the local container registry and the cluster are connected to the mirror registry's host, you can directly push the release images to the local registry.
+
+If multiple clusters are present within the restricted network, mirror the required release images to a single container image registry and use that registry to update all the clusters.
```

Comment:
This is one option, but I don't know if we have a Red Hat preference about how many mirror registries customers chose to use.  So we might want to soften the imperative to:

> A single container image registry may be sufficient to host mirrored images suppling several clusters in the disconnected network.

or some such, to explain that it is possible, but not required, to use a single mirror registry.

Reply:
I'll make this change now